tests/manage/z_cluster/nodes/test_node_kernel_crash_ceph_fsync.py

```diff
@@ -89,8 +88,7 @@ def test_node_kernel_crash_ceph_fsync(
         )
 
         file = constants.FSYNC
-        cmd = f"oc cp {file} {pvc_obj.namespace}/{pod_obj.name}:/"
-        helpers.run_cmd(cmd=cmd)
+        pod_obj.copy_to_pod_cat(file, "/")
```

Comment:
is `/` mount point writeable? inside pod

Reply:
I don't know )

I have replaced the line that will fail 100% because of existed `oc cp` limitations

cmd = f"oc cp {file} {pvc_obj.namespace}/{pod_obj.name}:/"
became 
pod_obj.copy_to_pod_cat(file, "/")

if it fails we fix it during Report Portal analysis